uperf_wrapper/trigger_uperf.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+import subprocess
+from datetime import datetime
+
+import elasticsearch
+import numpy as np
+
+
+class Trigger_uperf():
+    def __init__(self, args):
+        stdout = self._run_uperf(args.workload[0])
```

Comment:
I think all of this will need to move to a new function called emit_actions(), you can take a look at https://github.com/cloud-bulldozer/snafu/blob/master/run_snafu.py#L119 and that's why the comment below about yield documents and the index...